docs/case-studies/scheduling-blockages.md

```diff
@@ -0,0 +1,470 @@
+# Debugging scheduling blockages with tracing and callstack sampling
```

Comment:
Would be amazing if we could publicise this new article somehow. I really think it would be great if we could amplify this using a blogpost that would allow us to post it on Hacker News + similar.